src/components/PostsList.tsx

```diff
@@ -1,86 +1,65 @@
-import React from 'react';
+import { Post } from '../types/Post';
+import classNames from 'classnames';
 
-export const PostsList: React.FC = () => (
-  <div data-cy="PostsList">
-    <p className="title">Posts:</p>
+interface Props {
+  posts: Post[];
+  selectedPost: Post | null;
+  setSelectedpost: React.Dispatch<React.SetStateAction<Post | null>>;
```

Comment:
The prop `setSelectedpost` should be renamed to `setSelectedPost` to maintain consistency with the naming convention used for state variables and props.